src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -0,0 +1,231 @@
+<!--
```

Comment:
Just a FYI for future reference - when refactoring a large chunk of logic from one file to another, it is best to move the logic without any changes in one commit, and then make the actual changes in another.

Moving the logic and making changes all at once means reviewers and anyone looking at commit history has to go through this whole file to figure out what has changed.  As a reviewer, I basically have to re-review this whole file.